### DISCUSSION TITLE
Beware `master` branch `odinson` is not compatible with `master` branch `OdinsonWebapp`

### DISCUSSION THREAD

**Discussion by nezda** [BeckySharp/OdinsonWebapp#25] [General]
Took me a bit of fiddling around to realize the `master` branch of `OdinsonWebapp` is not compatible with the `master` branch of `odinson`. I indexed content using the `odinson` directions (on `master`) and then got all kinds of `sbt` and `scala` problems which seemed to get partially chewed up in the `OdinsonWebapp` error handling but eventually I worked out that I needed to re-index my content with `odinson` `val odinsonVer = "0.3.1"`. Checked out tag `v0.3.1` in `odinson`, re-indexed and all worked fine. I'm not sure if this warrants mention in the directions, or maybe its just a rare thing and most people use the docker setup, but I thought I'd mention.

Cheers,
- Luke

**Comment by BeckySharp** [comment]
Hmm.  I think I know what happened.... I think that likely https://github.com/lum-ai/odinson/pull/283 made the indexes incompatible, which means that (a) we should have bumped the major version and (b) we should re-release.  I've messaged ppl accordingly.  In the meantime I'll update the (scant) documentation :)

**Reply by BeckySharp** [reply]
addressed with commit d127cec
